wv2util/AppOverride.cs

```diff
@@ -217,6 +221,33 @@ public static void UpdateCollectionFromRegistry(ObservableCollection<AppOverride
                 entriesToRemove.Remove(entry);
             }
 
+            regKey = RegistryUtil.OpenRegistryPath(registryRoot, s_registryPathChannelSearchKind, false);
+            valueNames = regKey.GetValueNames();
+            foreach (string valueName in valueNames)
+            {
+              AppOverrideEntry entry = GetOrCreateEntry(appNameToEntry, collection, valueName, storageKind);
+              try
+              {
+                var value = regKey.GetValue(valueName);
+                int valueAsInt = (int)value;
+                entry.ReverseSearchOrder = 1 == valueAsInt;
+              }
+              catch (InvalidCastException)
+              {
+                Debug.WriteLine("Ignoring malformed registry entries that don't use an int: path=" + regKey + "." + valueName);

```

Comment:
Does this mean it doesn't work if its a string "1" only if its actually a number 1? Does the loader code work that way too?

Reply:
Updated to support both string and int